### PR TITLE
Update PR template

### LINKED ISSUE #1468: Can we skip final call on example updates?
The most common reason for us to go through a final call is that an example was updated. The original reason for us to do this was to allow implementors the time to check their implementation against the updated example. I'm not aware of anyone who does that. Certainly I haven't seen any feedback from an implementor who did this.

Most of the comments we get are editorial in nature. They are suggestions for improving the language. I think there is a mismatch between what we put into a final call, and what we are actually getting feedback on.

I bring this up, partially to revisit if we're doing final calls on the right things, and partially because I'd like to see if we can streamline the process a little more. There might be things we can do more efficiently without losing quality.

### PR DESCRIPTION
**Pull request by carlosapaduarte**
Update the PR template so that it's clear that editorial changes to the applicability, expectation or example sections do not require a Final Call period. Also, add a request for PR authors to leave the "How to Review and Approve" section in their PR description.

Closes issue(s):

- closes #1468 

Need for Final Call:
This will not require a Final Call 
---

## How to Review And Approve

- Go to the “Files changed” tab
- Here you will have the option to leave comments on different lines.
- Once the review is completed, find the “Review changes” button in the top right, select “Approve” (if you are really confident in the rule) or "Request changes" and click “Submit review”.
- Make sure to also review the proposed Final Call period. In case of disagreement, the longer period wins.

### CODE REVIEW

**Review by WilcoFiers — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -6,9 +6,10 @@ Closes issue(s):
 
 Need for Final Call:
 << choose one of the following and remove the rest >>
+<< leave the "How to Review and Approve" section in your pull request description >>
```
Comment (Jym77) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: Seems weird to have this as one of the "Need for Final Call" options.

**Review by Jym77 — CHANGES_REQUESTED** [review]

File: pages/design/process.md [hunk 2]
```diff
@@ -159,7 +159,7 @@ This includes, but might not be limited to changes that changes, extends or limi
 
 These changes are considered non-substantial and will not require a "Final call" before being published:
 
-- Editorial changes, that does not change the meaning of a rule (also if it's in the Applicability or Expectations)
+- Editorial changes, that does not change the meaning of a rule (also if it's in the Applicability, Expectations or Examples)
```
Comment (daniel-montalvo) [inline pages/design/process.md:162]:
```suggestion
- Editorial changes, that do not change the meaning of a rule (also if it's in the Applicability, Expectations or Examples)
```

**Review by daniel-montalvo — APPROVED** [review]
Just a small plural that I think should be there as we are speaking about changes.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (carlosapaduarte) [reply .github/PULL_REQUEST_TEMPLATE.md]: Indeed. Moved it to the instructions in the PR Etiquette section

**Review by Jym77 — APPROVED** [review]

**Comment by carlosapaduarte** [comment]
Update PR template approved. Merging

**Pull request merged**